internal/pkg/cli/app_init.go

```diff
@@ -254,48 +288,27 @@ This command is also run as part of "ecs-preview init".`,
 		Example: `
   Create a "frontend" web application.
   /code $ ecs-preview app init --name frontend --app-type "Load Balanced Web App" --dockerfile ./frontend/Dockerfile`,
-		PreRunE: runCmdE(func(cmd *cobra.Command, args []string) error {
-			opts.fs = &afero.Afero{Fs: afero.NewOsFs()}
-
-			store, err := store.New()
-			if err != nil {
-				return fmt.Errorf("couldn't connect to project datastore: %w", err)
-			}
-			opts.appStore = store
-			opts.projGetter = store
-
-			ws, err := workspace.New()
+		RunE: runCmdE(func(_ *cobra.Command, _ []string) error {
+			cmd, err := newInitAppCmd(opts)
```

Comment:
From our offline discussion, the `newInitAppCmd` taking in a mostly uninitialized `initAppOpts` struct feels off. I'm wondering if we can/should lean more on `viper` flag binding to decouple.

Reply:
Went with `viper` and also added a command factory to initialize commands for us